docs/source/merlin_server.rst

```diff
@@ -0,0 +1,70 @@
+Merlin Server
+=============
+The merlin server command allows users easy access to containerized broker 
+and results servers for merlin workflows. The main configuration will be
```

Comment:
Add a sentence after the first:
"This allows users to run merlin without a dedicated external server."
Then a blank line to have a new paragraph before "The main configuration"